Assets/Scripts/Copilots/HealingBubble.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealingBubble : MonoBehaviour
+{
+    //variables for spore particle system
+    public float sporeTimer;
+    private bool sporeStart;
+    private ParticleSystem spores;
+
+    //variables for healing bubble
+    private MeshRenderer bubbleMesh;
+    private Vector3 bubbleScale;
+    private bool bubbleInflated;
+
+    //variables for healing player
+    private bool healPlayer;
+    private float healTime;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        transform.localPosition = Vector3.zero;
```

Comment:
Not that this line has anything wrong with it but I was wondering why it was needed? Like if the prefabs transform is 0 then i don't think this is needed